### PR TITLE
movie header for dashboard title page

### PR DESCRIPTION
**Pull request by Mussuranae**
Movie header for dashboard

### CODE REVIEW

File: apps/financiers/financiers/src/app/dashboard/title/view/view.component.html [hunk 1]
```diff
@@ -2,52 +2,7 @@
...
-    <nav mat-tab-nav-bar>
-      <a mat-tab-link
-         *ngFor="let link of navLinks"
-         [routerLink]="link.path"
-         routerLinkActive #rla="routerLinkActive"
-         [active]="rla.isActive">
-        {{link.label}}
-      </a>
-    </nav>
-    <router-outlet></router-outlet>
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/dashboard/title/view/view.component.html:51]: clean also the scss file for this component?

**Review by pldespaigne — APPROVED** [review]
SGTM

File: apps/financiers/financiers/src/app/dashboard/tunnel/summary/summary.component.ts [hunk 2]
```diff
@@ -42,8 +42,8 @@ export class TunnelSummaryComponent {
         })
       } else {
         // Log the invalid forms
-        /* console.error(this.findInvalidControlsRecursive(this.form)) 
-      } 
+        /* console.error(this.findInvalidControlsRecursive(this.form))
+      }
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/dashboard/tunnel/summary/summary.component.ts:46]: euhm? remove or make a TODO?

**Review by fritzschoff — APPROVED** [review]

File: apps/financiers/financiers/src/app/dashboard/tunnel/summary/summary.component.ts (hunk 2 above)
Comment (Mussuranae) [reply apps/financiers/financiers/src/app/dashboard/tunnel/summary/summary.component.ts:46]: Yes, I will rework the summary page in the same time I do the migration script ;)

**Pull request merged**